txclib/wizard.py

```diff
@@ -0,0 +1,149 @@
+        are by default True when interactive wizard is run.
+
+        Returns: the populated (options, args) tuple.
+        """
+
+        TEXTS = messages.TEXTS
+        options, args = set_parser().parse_args([])
+        options.local = True
+        options.execute = True
+        logger.info(TEXTS['source_file']['description'])
+
+        ans = inquirer.prompt([
+            inquirer.Text('source_file',
+                          message=TEXTS['source_file']['message'],
+                          validate=validate_source_file)
+        ], raise_keyboard_interrupt=True)
+        options.source_file = ans['source_file']
+
+        logger.info(
+            TEXTS['expression']['description'].format(
+                source_file=options.source_file
+            )
+        )
+        ans = inquirer.prompt([
+            inquirer.Text('expression',
+                          message=TEXTS['expression']['message'],
+                          validate=validate_expression)
+        ], raise_keyboard_interrupt=True)
+        args.append(ans['expression'])
+
+        formats = self.get_formats(os.path.basename(options.source_file))
+        logger.info(TEXTS['formats']['description'])
+        ans = inquirer.prompt([
+            inquirer.List('i18n_type', message=TEXTS['formats']['message'],
+                          choices=formats)
+        ], raise_keyboard_interrupt=True)
+        options.i18n_type = ans['i18n_type']
+
+        organizations = self.get_organizations()
+        logger.info(TEXTS['organization']['description'])
+        org_answer = inquirer.prompt([
+            inquirer.List('organization',
+                          message=TEXTS['organization']['message'],
+                          choices=organizations)
+        ], raise_keyboard_interrupt=True)
+
+        projects = []
+        first_time = True
+        create_project = ("Create new project (show instructions)...",
+                          'tx:new_project')
+        project = None
+        while not project:
+            if not first_time:
+                inquirer.prompt([inquirer.Text(
+                    'retry',
+                    message="Hit Enter to try selecting a project again")
+                ], raise_keyboard_interrupt=True)
+
+            projects = self.get_projects_for_org(org_answer['organization'])
+            p_choices = [(p['name'], p['slug']) for p in projects]
+            if projects:
+                logger.info(TEXTS['projects']['description'])
+            else:
+                logger.info("We found no projects in this organization!")
+            first_time = False
+            ans = inquirer.prompt([
+                inquirer.List(
+                    'project',
+                    message=TEXTS['projects']['message'].format(self.host),
+                    choices=p_choices + [create_project]
+                )
+            ], raise_keyboard_interrupt=True)
+            if ans['project'] == 'tx:new_project':
```

Comment:
Are we sure no project of the user has these slugs? 😛

Reply:
Slugs don't allow `:`, so I think we are safe here :)